### DISCUSSION TITLE
Filament Theme Colors

### DISCUSSION THREAD

**Discussion by sc-hero** [filamentphp/filament#8241] [General]
Hello Filament Team,

I created theme manager resource, My theme manager have name, version, colors{"primary", "danger",...} so, In filament admin panel provider, How to use theme manager colors in colors array?

**Comment by leandrocfe** [comment]
You can use the `colors` method 
`$table->colors($colors)`
https://filamentphp.com/docs/3.x/panels/themes

**Reply by sc-hero** [reply]
->colors([
                'primary' => Theme::color(),
                'info' => "#fff",
                "warning" => "#000",
                "success" => "#000",
                "danger" => "#000"
            ])
            
            But show the error Call to a member function connection() on null, how to fix?

**Reply by leandrocfe** [reply]
I think you can do this
```php
->bootUsing(function () {
    FilamentColor::register(Theme::color());
})
```
